gemfiles/rails_6_1.gemfile

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+eval_gemfile "../Gemfile"
+
+if Gem::Version.new(RUBY_VERSION.dup) >= Gem::Version.new("3.1")
+  gem "net-imap", require: false
+  gem "net-pop", require: false
+  gem "net-smtp", require: false
```

Comment:
Should we specify versions for these gems?

Reply:
I don't think it's required.  I haven't seen any of the fixes for this issue in various repos add versions.